End_of_year_Reallocations/credit_test.py

```diff
@@ -42,21 +46,41 @@
         # get excldclosedacct flag
         .assign(excldclosedacct = lambda x: np.where((x['REPCODE'] == 'CL') & (x['lbu_py_tot'] == 0),'Y','N'))
 
+        # get cust level billing
```

Comment:
This is doing a lot in a single block of code. Why not add functions for doing particular parts of this meta-operation, like for example:

```python
def assign_customer_level_billing(obj):
  return obj
    .assign( # etc. )

def main():
  obj = # get my thing
  obj = assign_customer_level_billing(obj);
  # etc
```